versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 2.0"
+      version = ">= 4.6"
```

Comment:
This will also need to be bumped in the examples/complete directories for tests to pass